src/execution/operator/aggregate/physical_hash_aggregate.cpp

```diff
@@ -430,10 +463,14 @@ string PhysicalHashAggregate::ParamsToString() const {
 		result += groups[i]->GetName();
 	}
 	for (idx_t i = 0; i < aggregates.size(); i++) {
-		if (i > 0 || groups.size() > 0) {
+		auto &aggregate = (BoundAggregateExpression &)*aggregates[i];
+		if (i > 0 || !groups.empty()) {
 			result += "\n";
 		}
 		result += aggregates[i]->GetName();
+		if (aggregate.filter) {
+			result += aggregate.filter->GetName();
```

Comment:
Maybe add "FILTER " before this to the output, to make it clear that this is a filter op (similar to how it appears in a SQL statement).